src/opus.py

```diff
@@ -3,9 +3,15 @@
 import hashlib
 from typing import Any, Dict, List, Optional, Union
 from pathlib import Path
+import re
 import aiofiles
+import fitz
 import yaml
 
+LEFT_PAGE_LINE_NUMBER_END = (76, 77)
```

Comment:
These numbers should be in the opus instead (since it might be different for different pdfs, aight?)

Reply:
Good point, but it's also unfortunate if the person writing the opus (directly or through the future editor) has to measure it. I'll see if I can discover it automatically.

Reply:
I'll add a TODO instead, since we can live without it for now.